docs/drafts/index.html

```diff
@@ -7,13 +7,13 @@
 		<link rel="stylesheet" href="css/tables.css">
 		<script class="remove">
 			var respecConfig = {
-				group: "epub",
 				wgPublicList: "public-epub3",
+				latestVersion: "https://w3c.github.io/epub-tests/",
 				specStatus: "base",
 				noRecTrack: true,
-				publishDate: "2022-04-22",
+				publishDate: "2022-04-30",
 				shortName: "epub-tests",
-				edDraftURI: "https://w3c.github.io/epub-tests/results/",
+				edDraftURI: "https://w3c.github.io/epub-tests/results/drafts/",
                 copyrightStart: "2021",
 				editors:[
 					{
```

Comment:
Unrelated to this PR, but reminder that we were thinking of removing http://www.aptest.com/standards/htmldiff/htmldiff.pl (line 38) from all respec configs.

Reply:
You are right. I have removed these before merging.